pyaedt/modeler/Primitives.py

```diff
@@ -81,7 +81,7 @@ class Polyline(object):
 
     This class provides methods for creating and manipulating polyline objects within 
     the AEDT Modeler. Intended usage is for the constructor of this class to be called
-    by the ``Primitives.draw_polyline`` method. The documentation is provided there.
+    by the `Primitives.draw_polyline` method. The documentation is provided there.
```

Comment:
@maxcapodi78, I can find no mention of `draw_polyline`.  Did the method name change?